### ISSUE TITLE
Apple M1支持运行paddle Seg吗？

### ISSUE THREAD

**Issue by sy-l123** [PaddlePaddle/PaddleSeg#3656]
### 问题确认 Search before asking

- [X] 我已经查询[历史issue](https://github.com/PaddlePaddle/PaddleSeg/issues)(包括open与closed)，没有发现相似的bug。I have searched the [open and closed issues](https://github.com/PaddlePaddle/PaddleSeg/issues) and found no similar bug report.

### Bug描述 Describe the Bug

场景：运行paddle seg v2.8版本里面的python3 tools/train.py \
       --config configs/quick_start/pp_liteseg_optic_disc_512x512_1k.yml \
       --save_interval 500 \
       --do_eval \
       --use_vdl \
       --save_dir output

输出：
2024-02-22 23:03:08 [INFO]      Use the following config to build optimizer
optimizer:
  momentum: 0.9
  type: Momentum
  weight_decay: 4.0e-05
2024-02-22 23:03:08 [INFO]      Use the following config to build loss
loss:
  coef:
  - 1
  - 1
  - 1
  types:
  - type: CrossEntropyLoss
  - type: CrossEntropyLoss
  - type: CrossEntropyLoss
tests/train.sh: line 6: 57086 Segmentation fault: 11  python3 tools/train.py --config configs/quick_start/pp_liteseg_optic_disc_512x512_1k.yml --save_interval 500 --do_eval --use_vdl --save_dir output

output只有vdlrecords.1708614188.log文件输出；请问是哪里出错了

### 复现环境 Environment

Macos ：13.6.3 (22G436)，Apple M1
paddlepaddle：2.4.2
paddleseg：2.8

### Bug描述确认 Bug description confirmation

- [X] 我确认已经提供了Bug复现步骤、代码改动说明、以及环境信息，确认问题是可以复现的。I confirm that the bug replication steps, code change instructions, and environment information have been provided, and the problem can be reproduced.

### 是否愿意提交PR？ Are you willing to submit a PR?

- [ ] 我愿意提交PR！I'd like to help by submitting a PR!

**Comment by shiyutang** [comment]
你好，这个日志看上去并没有任何的报错，如果这是所有信息的话，那么可能是训练时间不够长导致的没有模型文件输出。

**Comment by sy-l123** [comment]
> 你好，这个日志看上去并没有任何的报错，如果这是所有信息的话，那么可能是训练时间不够长导致的没有模型文件输出。

paddlepaddle换了最新版本已经跑起来了，一直运行：I0228 11:19:19.739109 3918831872 kernel_dispatch.h:102] Get BackendSet from tensor是不是就是在训练了？请问需要多久啊？

**Comment by shiyutang** [comment]
可以设置log iter 比较小，例如100 ，因此就能很快看到训练的日志。

**Comment by sy-l123** [comment]
> 可以设置log iter 比较小，例如100 ，因此就能很快看到训练的日志。

好的，感谢，已经成功。
